device_parser.c

```diff
@@ -49,7 +48,7 @@ static void nextsym(struct device_parser *dp)
 	}
 }
 
-static int accept(struct device_parser *dp, int type, char *scalar)
+int accept(struct device_parser *dp, int type, char *scalar)
```

Comment:
Please add a useful prefix to these funcions. In the end, there is already `accept(3)` function / syscall which does a different thing.

Reply:
Ack